contracts/vaults/TreasuryVault/RibbonTreasuryVault.sol

```diff
@@ -346,12 +341,11 @@ contract RibbonTreasuryVault is
         for (uint256 i = 0; i < _initParams._whitelist.length; i++) {
             require(_initParams._whitelist[i] != address(0), "Whitelist null");
```

Comment:
To refactor it, what you could do is to create a `_addWhitelist` function to share code between initialize and `addWhitelist`. Example:

```solidity

function initialize() {
  _addWhitelist()
}

function addWhitelist() {
  _addWhitelist()
}

function _addWhitelist() {
}
```